cpp/ps0005/ps0005_solution.cpp

```diff
@@ -63,29 +58,31 @@ int main()
         else {
             cout << "(" << num1 << ") / (" << num2 << ") = " << num1 / num2 << "" <<endl;
         }
-    }
+    } // 1/1
 
-    else if (op == '%')
+    else if (op == '%')
     {
-        if (floor(num1) != num1 || floor(num2) != num2)
+        if (floor(num1) != num1 || floor(num2) != num2)
         {
             cout << "cannot execute modulo operation for floating point numbers." <<endl;
         }
-        else if (num2 == 0)
+        else if (num2 == 0)
         {
             cout << "Math Error: cannot perform modulo with 0." <<endl;
         }
         else {
+            // printing the result if all data is valid 1/1
             int result = static_cast<int>(num1) % static_cast<int>(num2);
             cout << "(" << static_cast<int>(num1) << ") % (" << static_cast<int>(num2) << ") = " << result << "" <<endl;
         }
-
-    }
+    } // 1/1
 
     else 
     {
         cout << "Invalid character for operator." <<endl;
-    }
+    } // 1/1
+    // 1/1
 

```

Comment:
- [ ] used `switch` and `if` structures in a right way to handle different cases
- [x] handled division operation `/`
- [x] handled modulo operation `%`
- [x] handled invalid operator case
- [x] chose the right operation for each case
- [x] printed the result of the chosen operation